packages/imperative/src/cmd/__tests__/CommandProcessor.unit.test.ts

```diff
@@ -25,6 +25,8 @@ import { ImperativeConfig } from "../../utilities/src/ImperativeConfig";
 import { setupConfigToLoad } from "../../../__tests__/src/TestUtil";
 import { EnvFileUtils } from "../../utilities";
 import { join } from "path";
+import { Config } from "../../config";
+import { LoggerUtils } from "../..";
```

Comment:
This needs to be imported with the full path, since it was removed from exports in V3.